kvui.py

```diff
@@ -152,10 +152,10 @@ def get_text(self):
                     text += f"\nA new !hint <itemname> costs {ctx.hint_cost}% of checks made. " \
                             f"For you this means every " \
                             f"{max(min_cost, int(ctx.hint_cost * 0.01 * ctx.total_locations))}" \
-                            f" location checks."
+                            "location checks."
+                    text += f"\nYou currently have {ctx.hint_points} points."
```

Comment:
Might be better to concat this into the previous append operation rather than create a new one.